File: my-folder/0027-remove-element/solution.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int res = 0;
        for (int i = 0; i < nums.length; i++) {
            if (nums[i] != val) {
                int temp = nums[res];
                nums[res] = nums[i];
                nums[i] = temp;
                res++; 
            }
        }
        return res;
    }
}

File: my-folder/0027-remove-element/solution.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        res = 0
        left = 0
        right = len(nums) - 1
        while left <= right:
            if nums[left] == val:
                nums[left], nums[right] = nums[right], nums[left]
                right -= 1
            else:
                res += 1
                left += 1
        return res
